### PR TITLE
Remove date formatting from validFrom/expiry

### PR DESCRIPTION
**Pull request by tylerwoonton**
Remove the automatic date formatting from the validFrom and expiry m/y fields.

### CODE REVIEW

File: tests/Billing/PaymentCardClientTest.php [hunk 1]
```diff
@@ -50,8 +51,8 @@ public function get_payment_cards_by_id()
         $this->assertEquals("M44 0AB", $paymentCard->postcode);
         $this->assertEquals("4111111111111111", $paymentCard->cardNumber);
         $this->assertEquals("Visa", $paymentCard->cardType);
-        $this->assertEquals('2019-10-31T00:00:00+0000', $paymentCard->validFrom->format(DateTime::ISO8601));
-        $this->assertEquals('2035-10-31T00:00:00+0000', $paymentCard->expiry->format(DateTime::ISO8601));
+        $this->assertEquals('2019-10-31T00:00:00+0000', (new DateTime($paymentCard->validFrom))->format(DateTime::ISO8601));
```
Comment (Gman98ish) [inline tests/Billing/PaymentCardClientTest.php]: Wait, should this be in the format ISO8601 or `m-y`?

Comment (tylerwoonton) [reply tests/Billing/PaymentCardClientTest.php]: Should be m/y will resolve shortly.

File: src/Billing/Entities/PaymentCard.php [hunk 2]
```diff
@@ -13,18 +13,18 @@
...
  */
 class PaymentCard extends Entity
 {
-    protected $dates = ['validFrom', 'expiry'];
+    protected $dates = [];
 
     public function isExpired()
     {
-        return $this->expiry < new DateTime();
+        return DateTime::createFromFormat('m/y', $this->expiry) < new DateTime();
```
Comment (rbibby) [inline src/Billing/Entities/PaymentCard.php]: Is there any tests for this function? If not would be good to add some

File: src/Billing/Entities/PaymentCard.php [hunk 3]
```diff
@@ -13,18 +13,18 @@
...
- * @property DateTime $expiry
+ * @property string $validFrom
+ * @property string $expiry
  * @property int $issueNumber
  * @property boolean $primaryCard
  */
 class PaymentCard extends Entity
 {
-    protected $dates = ['validFrom', 'expiry'];
+    protected $dates = [];
```
Comment (rbibby) [inline src/Billing/Entities/PaymentCard.php]: This can be removed if it's not needed anymore

File: src/Billing/Entities/PaymentCard.php [hunk 4]
```diff
@@ -13,20 +13,32 @@
...
+     *
+     * @return bool
+     * @throws \Exception
+     */
     public function isExpired()
     {
-        return $this->expiry < new DateTime();
+        $date = DateTime::createFromFormat('m/y', $this->expiry);
+
+        return $date->format('Y-m-t') < (new DateTime())->format('Y-m-d');
```
Comment (Gman98ish) [inline src/Billing/Entities/PaymentCard.php]: Why are you comparing them as strings?

Surely you want to compare them as DateTime objects?

**Pull request merged**